src/labels/CustomLabels.labels

```diff
@@ -2028,7 +2028,7 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>healthSolutionAccountModel</shortDescription>
-        <value>All of your individual Contacts should use the same Account Model. View these Contacts by running the &lt;b&gt;Individual Contacts by Account Type&lt;/b&gt; report in the NPSP Health Check reports folder.</value>
+        <value>All of your Accounts should use the same Account Model. View these Accounts by running the &lt;b&gt;Individual Accounts by Account Type&lt;/b&gt; report in the NPSP Health Check reports folder.</value>
```

Comment:
@dospromptman @davidhabib 

The Account Model is set to Household Account, but Health Check has found that you're using a mixture of Account types.

All of your Accounts should use the same Account Model. View all of your Accounts and their types by running the **Individual Accounts by Account Type** report in the NPSP Health Check reports folder.